cilium-dbg/cmd/bpf_ct_list.go

```diff
@@ -23,10 +23,9 @@ import (
 // bpfCtListCmd represents the bpf_ct_list command
 var (
 	bpfCtListCmd = &cobra.Command{
-		Use:     "list ( global | endpoint | cluster ) [identifier]",
+		Use:     "list ( global | cluster ) [identifier]",
```

Comment:
nit: maybe `list ( global | cluster <identifier> )` - now that only `cluster` expects an identifier

Reply:
I would use `<...>` - because `[...]` indicate optional arguments (cobra comment on the `Use`) - and it's no longer optional in case of `cluster`